bin/install-wp-tests.sh

```diff
@@ -179,6 +179,12 @@ install_deps() {
 	# As zip file does not include tests, we have to get it from git repo.
 	git clone --depth 1 --branch $LATEST_WC_TAG https://github.com/woocommerce/woocommerce.git
 
+	# Bring in WooCommerce Core dependencies
+ 	composer self-update --1
```

Comment:
A downgrade of Composer is required since the latest release of WooCommerce is not compatible with version 2.